corehq/apps/geospatial/forms.py

```diff
@@ -45,7 +46,32 @@ def __init__(self, *args, **kwargs):
                         'user_location_property_name',
                         data_bind="value: customUserFieldName"
                     ),
-                    crispy.Field('case_location_property_name', data_bind="value: geoCasePropertyName"),
+                    crispy.Field(
+                        'case_location_property_name',
+                        data_bind="options: geoCasePropOptions, "
+                                  "value: geoCasePropertyName, "
+                                  "event: {change: onGeoCasePropChange}"
+                    ),
+                    crispy.Div(
+                        crispy.HTML('%s' % _(
+                            "The custom case property has been deprecated in the Data Dictionary. "
```

Comment:
nit: Can probably leave out "custom". I don't think users understand what that means.

Are we still showing the deprecated case property in the dropdown? or it appears blank?

Reply:
Good point, I'll change the wording. Once the user has changed properties and clicked save then the deprecated case property that was selected will no longer appear in the dropdown.

Reply:
But what happens when they visit this page after the case property was deprecated? they see an empty drop down selection or they do see the deprecated case property?

Reply:
Ah, after testing this again I see that we show deprecated GPS properties in the dropdown. So if the user has deprecated the selected case property it will still be in the dropdown. I'll need to make a PR to update this so that all but the selected deprecated property doesn't show in the dropdown.

Reply:
> So if the user has deprecated the selected case property it will still be in the dropdown

I believe that is what should happen. The user should see what they have set for now, and then update to something non deprecated.
Is this on staging somewhere to see?

Reply:
Yes, these changes can be seen on [this](https://staging.commcarehq.org/a/test-65/geospatial/settings/) staging domain.

Reply:
I have created a PR to hide deprecated case properties [here](https://github.com/dimagi/commcare-hq/pull/33454).